### PR TITLE
Added the certificate_logo_image_url field to the Organization model

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6858

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -162,6 +162,10 @@ class Organization(TimeStampedModel):
     homepage_url = models.URLField(max_length=255, null=True, blank=True)
     logo_image_url = models.URLField(null=True, blank=True)
     banner_image_url = models.URLField(null=True, blank=True)
+    certificate_logo_image_url = models.URLField(
+        null=True, blank=True, help_text=_('Logo to be displayed on certificates. If this logo is the same as '
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/models.py:166]: I'm in favor of making this field required. Does it mean we don't need to perform the check we discussed at render-time?

**Review by rlucioni — COMMENTED** [review]
Looks like you need to add a migration.

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (clintonb) [reply course_discovery/apps/course_metadata/models.py:166]: We cannot make this required. Not all organizations have certificates.

Comment (rlucioni) [reply course_discovery/apps/course_metadata/models.py:166]: My bad, ignore me. I didn't see `blank=True`.

**Comment by clintonb** [comment]
@edx/ecommerce please review

**Review by rlucioni — APPROVED** [review]

**Comment by MatthewPiatetsky** [comment]
👍 after rebase

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**